t.py

```diff
@@ -52,8 +52,7 @@ def querys(session, user):
 
                 data = json.loads(r.text)
                 for d in data['scores']['scores']:
-                    if(u"通識" in d['title'] and
-                       float(d['final_score']) >= 60.0):
+                    if '通識' in d['title'] and d['final_score'] != '' and float(d['final_score']) >= 60.0:
```

Comment:
I assume the line is over 80 characters, please wrap this condition

Reply:
```python
if ('通識' in d['title'] and d['final_score'] != '' and
        float(d['final_score']) >= 60.0):
```
Like this?

Reply:
yep